### PR TITLE
Define CRM

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
## Description

Update categories.md:

- [x] Define CRM

## Type of suggestion

* [x] Styleguide

## Product & version (if relevant)
Marketplace integrations

### CODE REVIEW

File: marketplace-and-integrations/integrations/categories.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Umbraco HQ has developed and made available integrations with different SaaS (So
...
...a></td></tr><tr><td><strong>ActiveCampaign with Umbraco Forms</strong></td><td>A custom workflow that allows form entries to be mapped to an ActiveCampaign contact and is stored within the Customer Relationship Management (CRM) platform.</td><td><a href="activecampaign.md">activecampaign.md</a></td><td><a href="../.gitbook/assets/activecampaign (1).png">activecampaign (1).png</a></td></tr><tr><td><strong>Dynamics</strong></td><td>A form picker and rendering component for Dynamics 365 forms.</td><td><a href="dynamics.md">dynamics.md</a></td><td><a href="../.gitbook/assets/dynamics (1).png">dynamics (1).png</a></td></tr><tr><td><strong>Hubspot</strong></td><td>A form picker and rendering component for Hubspot forms.</td><td><a href="hubspot.md">hubspot.md</a></td><td><a href="../.gitbook/assets/hubspot.png">hubspot.png</a></td></tr><tr><td><strong>Hubspot with Umbraco Forms</strong></td><td>A custom workflow that allows form entries to be mapped to a HubSpot contact record, and then stored within the CRM platform.</td><td><a href="hubspot-with-umbraco-forms.md">hubspot-with-umbraco-forms.md</a></td><td><a href="../.gitbook/assets/hubspot.png">hubspot.png</a></td></tr></tbody></table>
```
Comment (alina-tincas) [inline marketplace-and-integrations/integrations/categories.md:25]: warning shows that it needs to be defined on last sentence as well 💡 
```suggestion
<table data-card-size="large" data-view="cards"><thead><tr><th></th><th></th><th data-hidden data-card-target data-type="content-ref"></th><th data-hidden data-card-cover data-type="files"></th></tr></thead><tbody><tr><td><strong>ActiveCampaign</strong></td><td>A form picker and rendering component for ActiveCampaign forms.</td><td><a href="activecampaign.md">activecampaign.md</a></td><td><a href="../.gitbook/assets/activecampaign (1).png">activecampaign (1).png</a></td></tr><tr><td><strong>ActiveCampaign with Umbraco Forms</strong></td><td>A custom workflow that allows form entries to be mapped to an ActiveCampaign contact and is stored within the Customer Relationship Management (CRM) platform.</td><td><a href="activecampaign.md">activecampaign.md</a></td><td><a href="../.gitbook/assets/activecampaign (1).png">activecampaign (1).png</a></td></tr><tr><td><strong>Dynamics</strong></td><td>A form picker and rendering component for Dynamics 365 forms.</td><td><a href="dynamics.md">dynamics.md</a></td><td><a href="../.gitbook/assets/dynamics (1).png">dynamics (1).png</a></td></tr><tr><td><strong>Hubspot</strong></td><td>A form picker and rendering component for Hubspot forms.</td><td><a href="hubspot.md">hubspot.md</a></td><td><a href="../.gitbook/assets/hubspot.png">hubspot.png</a></td></tr><tr><td><strong>Hubspot with Umbraco Forms</strong></td><td>A custom workflow that allows form entries to be mapped to a HubSpot contact record, and then stored within the Customer Relationship Management (CRM) platform.</td><td><a href="hubspot-with-umbraco-forms.md">hubspot-with-umbraco-forms.md</a></td><td><a href="../.gitbook/assets/hubspot.png">hubspot.png</a></td></tr></tbody></table>
```

**Review by alina-tincas — CHANGES_REQUESTED** [review]
Thank you for the PR @erikjanwestendorp #H5YR 

I can see the warning shows that CRM needs to be defined on the last sentence as well. Can you please look into it? 😊

File: marketplace-and-integrations/integrations/categories.md (hunk 1 above)
Comment (erikjanwestendorp) [reply marketplace-and-integrations/integrations/categories.md:25]: @alina-tincas Thanks for your review, of course I can define it there as well, but I thought it was only necessary for the first time use? Please let me know 😄 

https://github.com/erikjanwestendorp/UmbracoDocs/blob/main/generic/contribute/style-guide.md#defined-acronyms

Comment (alina-tincas) [reply marketplace-and-integrations/integrations/categories.md:25]: That is correct, it is only necessary for first-time use, however, it seems that Vale still gives a warning if you check here: https://github.com/umbraco/UmbracoDocs/pull/5456/files. My thinking is that perhaps this is a table so it doesn´t catch the first-time definition 🤔

Comment (alina-tincas) [reply marketplace-and-integrations/integrations/categories.md:25]: However I can still merge it in as it is 👍

**Review by alina-tincas — APPROVED** [review]

File: marketplace-and-integrations/integrations/categories.md (hunk 1 above)
Comment (alina-tincas) [reply marketplace-and-integrations/integrations/categories.md:25]: As I can see on the documentation: https://docs.umbraco.com/umbraco-dxp/integrations/categories it seems that CRM is defined at the top of the cards 🤔 
image

**Review by alina-tincas — CHANGES_REQUESTED** [review]

File: marketplace-and-integrations/integrations/categories.md (hunk 1 above)
Comment (alina-tincas) [reply marketplace-and-integrations/integrations/categories.md:25]: so, in this case, it shouldn´t be necessary to make this change to add the definition for the "ActiveCampaign" card and/or "Hubspot". What do you think about this @erikjanwestendorp? 💡

Comment (erikjanwestendorp) [reply marketplace-and-integrations/integrations/categories.md:25]: @alina-tincas I think we can close this one 😄 Add this to all cards feels a bit to much of a good thing right?

**Comment by erikjanwestendorp** [comment]
@alina-tincas Closing this one since CRM is already defined! 😄

File: marketplace-and-integrations/integrations/categories.md (hunk 1 above)
Comment (alina-tincas) [reply marketplace-and-integrations/integrations/categories.md:25]: Agreed 😊

**Pull request closed without merging**